README.md

```diff
@@ -1,2 +1,18 @@
-# eslint
-Different technologies specific linting rules for ESLint
+# Introduction
+ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
```

Comment:
"for identifying and reporting" can be "to identify and report"
"of making code more consistent and avoiding bugs" can be "to make code more consistent and avoid bugs"

Reply:
Updated